packages/builder/src/components/design/settings/controls/ColumnEditor/ColumnDrawer.svelte

```diff
@@ -110,6 +111,7 @@
             {#each columns as column (column.id)}
               <div class="column" animate:flip={{ duration: flipDurationMs }}>
                 <div
+                  hide={!allowReorder || null}
```

Comment:
Custom HTML attributes should always be prefixed with `data-` per the HTML5 spec. Querying by custom attributes is also more expensive than class names, so we may as well stick with the normal approach of using classes and just update this to `class:hide={!allowReorder}` (we can remove the `|| null` as well).

Reply:
Good to know - thanks! Updated.